plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanSession.kt

```diff
@@ -267,18 +274,21 @@ class CodeWhispererCodeScanSession(val sessionContext: CodeScanSessionContext) {
         )
         createUploadUrlResponse
     } catch (e: Exception) {
-        LOG.error { "Security scan failed. Something went wrong uploading artifacts: ${e.message}" }
-        uploadArtifactFailedError()
+        throw e
```

Comment:
Why are we catching and throwing this exception?

Reply:
Removed try catch block here.